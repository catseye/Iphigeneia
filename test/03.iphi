(* Test nested 'var ... in ...' and arithmetic operators *)

var i in var j in var k in begin
    i := 2;
    j := 3;
    k := i + j;   (*  5 *)
    i := j * k;   (* 15 *)
    j := i / 2;   (*  7 *)
    j := j - 1    (*  6 *)
end
