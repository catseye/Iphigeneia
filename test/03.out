j=6
i=15
k=5
